osu.Game/Rulesets/Edit/SnapType.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+
+namespace osu.Game.Rulesets.Edit
+{
+    [Flags]
+    public enum SnapType
+    {
+        NearbyObjects = 0,
+        Grids = 1,
+        All = NearbyObjects | Grids,
```

Comment:
I mean zero is just never a valid flag enum value.

Also let's do `1, 1 << 1, 1 << 2` rather than raw values please.

Reply:
> Also let's do `1, 1 << 1, 1 << 2` rather than raw values please.

I was going to do this, but all existing ones were specifying values so I followed suite.